conftest.py

```diff
@@ -5,6 +5,11 @@
 
 from __future__ import annotations
 
+try:
+    import readline  # required to prevent a segfault on Python 3.10
```

Comment:
A link to the open issue is better https://github.com/moderngl/moderngl/issues/517
```suggestion
    # https://github.com/moderngl/moderngl/issues/517
    import readline  # required to prevent a segfault on Python 3.10
```